apps/client-server/src/app/post-parsers/post-parsers.service.ts

```diff
@@ -58,7 +58,7 @@ export class PostParsersService {
           instance,
           defaultOpts,
           websiteOpts,
-          tags,
+          websiteOpts.joinTags(tags),
```

Comment:
issue: this feels like a unnecessary optimization and code-bloat just to add the joinTags which is only used in this one specific spot. The only true acceptable way to inject tags as string into a description would be space separated.

I am not really seeing much of a benefit here.